index.html

```diff
@@ -12521,8 +12524,10 @@ <h2>Keywords</h2>
       The <code>@index</code> keyword MAY be aliased and MAY be used as a key in a
       <a>node object</a>, <a>value object</a>, <a>graph object</a>, <a>set object</a>, or <a>list object</a>.
       Its value MUST be a <a>string</a>.
-      <p>The unaliased <code>@index</code> MAY be used as the value of the <code>@container</code> key within an <a>expanded term definition</a>.</p>
-      <p>See <a class="sectionRef" href="#index-maps"></a> for a further discussion.</p>
+      <p>The unaliased <code>@index</code> MAY be used as the value of the <code>@container</code> key within an <a>expanded term definition</a>
+        and as an entry in a <a>expanded term definition</a>, where the value is `true` or `false`.</p>
```

Comment:
Same as above...

Reply:
Fixed in 8d3d281085220e9825afb419a809359d9380354e.